composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "bitbull/magento2-module-instagramwidget",
   "description": "Instagram stream for Magento2",
   "license": "proprietary",
+  "version": "1.0.1",
```

Comment:
Ok but I can't just specify it in the composer.json instead of keeping aligned the tag in two points of the code?